spec.bs

```diff
@@ -3792,6 +3826,30 @@ step "Set [=this=]'s [=Request/request=] to |request|":
 
 </div>
 
+<div algorithm="iframe navigation capture adAuctionHeaders attribute patch">
+Modify the <{iframe}> element to add a
+<dfn element-attr for="iframe">adAuctionHeaders</dfn> <a spec=html>content attribute</a>.

```

Comment:
Content attributes are all lowercase; see https://html.spec.whatwg.org/C#dom-link-fetchpriority as an example to go off of, for a new IDL/content attribute combo. Also we should decide whether we actually need the content attribute in this case or not. Note that the link element's disabled IDL attribute reflects the content attribute of the same name, whereas the style element's disabled IDL attribute _does not_.

I'm currently inquiring about the principles that would help us decide whether to reflect, or not (i.e., to have a content attribute or not). I'll report back when I hear from some others that might know better.

Reply:
I guess one thing we could use to guide us here is knowing how the attribute is intended to be used. Is it meant to be used strictly as a JS API (via an IDL attribute)? Or do you expect people to use it via HTML as well. My hunch is this should be a content & IDL attribute, in which case our approach here is mostly fine.

Reply:
I've changed the content attribute here and elsewhere to be all-lowercase.

In terms of how this is used, it is expected that people will use this via HTML as well as via the JS API. Thanks!